src/app.html

```diff
@@ -3,6 +3,8 @@
 	<head>
 		<meta charset="utf-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png" />
+    	<link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png" />
```

Comment:
this line doesn't align with the others
```suggestion
		<link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png" />
```